src/sandstorm/spk.c++

```diff
@@ -918,12 +1006,66 @@ private:
           "  # `spk dev` will write a list of all the files your app uses to this file.\n"
           "  # You should review it later, before shipping your app.\n"
           "\n"
-          "  alwaysInclude = []\n"
+          "  alwaysInclude = [],\n"
           "  # Fill this list with more names of files or directories that should be\n"
           "  # included in your package, even if not listed in sandstorm-files.list.\n"
           "  # Use this to force-include stuff that you know you need but which may\n"
           "  # not have been detected as a dependency during `spk dev`. If you list\n"
           "  # a directory here, its entire contents will be included recursively.\n",
+          "\n"
+          "  #bridgeConfig = (\n"
+          "  #  # Used for integrating permissions and roles into the Sandstorm shell\n"
+          "  #  # and for sandstorm-http-bridge to pass to your app.\n"
+          "  #  # Uncomment this block and adjust the permissions and roles to make\n"
+          "  #  # sense for your app.\n"
+          "  #  # For more detailed information, see \"BridgeConfig\" in\n"
+          "  #  # https://github.com/sandstorm-io/sandstorm/blob/master/src/sandstorm/package.capnp\n"
```

Comment:
Point to the authentication docs rather than `package.capnp`?

https://docs.sandstorm.io/en/latest/developing/auth/